docs/realm-selection.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Realm Selection"
+slug: "/contributor/catalyst/realm-selection"
+---
+
+A [Catalyst](https://github.com/decentraland/catalyst) is a Server that bundles different Services required by the Decentraland World. Some of the main responsibilities of the server are the management of the decentralized storage for most of the content needed by the client and the orchestration of communications between peers.
+
+When entering into a Decentraland Client, then you need to connect to an specific Catalyst for the backend services support: Content, Lambdas & Comms. One of the DAO Catalysts needs to be used, it could be picked randomly but that could lead to the user to access a backend service with great latency or a realm with a low amount of users while they are connected to another one.
```

Comment:
```suggestion
When entering into a Decentraland Client, then you need to connect to a specific Catalyst for the backend services support Content, Lambdas & Comms. One of the DAO Catalysts needs to be used, it could be picked randomly but that could lead to the user accessing a backend service with great latency or a realm with a low amount of users while they are connected to another one.
```